src/vos/sys_db.c

```diff
@@ -416,6 +429,8 @@ vos_db_init_ex(const char *db_path, const char *db_name, bool force_create, bool
 	}
 	return 0;
 failed:
+	if (attr != ABT_MUTEX_ATTR_NULL)
+		ABT_mutex_attr_free(&attr);
```

Comment:
Looks the 'attr' is leaked on success case, the 'attr' should be immediately freed after the mutex is created.

Reply:
Right, refreshed the patch for fix.